libcloud/container/drivers/gke.py

```diff
@@ -0,0 +1,156 @@
+from libcloud.common.google import GoogleOAuth2Credential
+
+from libcloud.container.drivers.docker import (DockerContainerDriver,
+                                               DockerConnection)
+
+
+class GKEContainerDriver(DockerContainerDriver):
```

Comment:
Will you inherit from the KubernetesContainerDriver class? Also check out the container driver type enum. You want the clustered one.